python/ray/experimental/sgd/tensorflow/distributed_tensorflow_runner.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import logging
+import os
+import json
+import tensorflow as tf
+
+from ray.experimental.sgd.tensorflow.tensorflow_runner import TensorFlowRunner
+
+logger = logging.getLogger(__name__)
+
+
+class DistributedTensorFlowRunner(TensorFlowRunner):
```

Comment:
Is the inheritance necessary here?

Reply:
class TensorFlowRunner's method get_state and set_state can be used to get current model. So I think inheritance is needed here.

Reply:
I think you should just define `get_state` and `set_state` in this class, so I think we don't need a separate DistributedTensorFlowRunner clas - just a regular TensorFlowRunner.